core/base/src/TString.cxx

```diff
@@ -131,6 +131,9 @@ TString::TString(const char *cs, Ssiz_t n)
       Zero();
       return;
    }
+   if (cs.find("\0" != std::string::npos)) {
```

Comment:
This cannot compile (calling `find` on a `const char *`)

Reply:
Ah ok. How can I fix this without using std::string() ?

Reply:
You can for instance loop over the `cs` array from 0 to n-1.